src/background/pageaction.ts

```diff
@@ -45,17 +45,19 @@ export class PageAction {
       );
       color = container.color;
     }
-    browser.pageAction.setIcon({
-      path: {
-        '19': `icons/pageaction-${color}-19.svg`,
-        '38': `icons/pageaction-${color}-38.svg`,
-      },
-      tabId: activatedTab.id,
-    });
-    if (!this.pref.pageAction) {
-      browser.pageAction.hide(activatedTab.id);
-    } else {
-      browser.pageAction.show(activatedTab.id);
+    if (activatedTab && activatedTab.id) {
```

Comment:
```suggestion
    if (activatedTab?.id) {
```

Reply:
Wow - that;s some new typescript syntax I haven't seen before.  It sorta goes against my old-school verbosity...  but I'll adopt it!

Reply:
Yeah, TS is a bit ahead of time again by already supporting [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), but that'll land in JS soon(tm) as well